release-notes/opensearch-dashboards-maps.release-notes-2.7.0.0.md

```diff
@@ -0,0 +1,51 @@
+## Version 2.7.0.0 Release Notes
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.7.0
+
+### Features
+* Add geo shape query filter ([#319](https://github.com/opensearch-project/dashboards-maps/pull/319))
+* Support adding static label to document layer ([#322](https://github.com/opensearch-project/dashboards-maps/pull/322))
```

Comment:
Could you help add the PR here? This one was missed before CHNAGELOG onboarding. https://github.com/opensearch-project/dashboards-maps/pull/336